source/macos/AdaptiveCards/AdaptiveCards/Base/Targets.swift

```diff
@@ -19,6 +19,25 @@ class ActionOpenURLTarget: NSObject, TargetHandler {
         delegate?.handleOpenURLAction(button: sender, urlString: url)
     }
 }
+    
+class ActionSubmitTarget: NSObject, TargetHandler {
+    weak var delegate: TargetHandlerDelegate?
+    let dataJson: String
```

Comment:
```suggestion
    let dataJson: String?
```
We can have this as optional